sdk-docs/maintaining-your-app.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Maintaining Your Application
+category: 61fcd8e1a448f5004215317c
+parentDocSlug: iris-app-sdk-reference
+---
+
+> 🚧 Beta
+> 
+> Please note that this is a beta version and is subject to change without notice. The final version may include changes to pricing, terms, conditions, and availability.
+
+### Security Patching
+As a developer, it is essential to keep your application and its dependencies up-to-date, ensuring that there are no vulnerabilities. With each update, you should bump the version number and submit it for review. Our team will manually review and approve each patch, ensuring the safety and stability of the platform. We are continiously scanning apps on the platform for vulnurabilities and not upgrading your app can cause it to be disabled.
+
+### Upgrading SDK Packages
+We will periodically release updates for our SDK libraries. It is crucial to keep your applications using the latest versions to maintain compatibility as our platform evolves. You may receive a warning in the terminal when publishing your app if your SDK is outdated. While you can initially ignore this warning, outdated SDK versions may eventually result in your app not being approved or even removed from the platform.
+
+### Updating Your Nx Workspace
+The SDKs rely on specific Nx and Node.js versions. Ensure that your application complies with the required versions at all times. Do *not* update your Nx workspace immediately when a new version is released. Instead, only upgrade when our SDKs depend on a newer version.
```

Comment:
It's difficult to know when our packages depend on a new NX version. But the prerequisites page will be updated and warnings will tell.